README.md

```diff
@@ -2,20 +2,31 @@
 
 Starter project for a chess-based TDD exercise.
 
-## Running the application
+## Getting the application up and running
 
-To run the application, first ensure that you have installed Poetry on your system. Install
-dependencies using the command ``poetry install``.
+### Installing dependencies
+First, let's ensure that you have installed python and pip on your system. 
 
-To run the application, use the command ``poetry run flask run`` and navigate to http://localhost:5000. You can reset the board at any time by restarting the server.
+- Run `python --version` and you should see a version number. If not, you may need to install Python on your machine. On ACG, this should be already available - ensure you're using the image specified in the instructions.
```

Comment:
Are we assuming that they should just be able to set up the ACG server and clone the repo down onto it at this stage?
(I think that's fine, as long as there's no particular settings for ACG, e.g. the smallest size should suffice)

Reply:
I think so - without needing to compile poetry and such the smallest size is now fine. I don't think there's any fiddling to do beyond what the instructions specify (choose Ubuntu X.Y, run sudo apt install python3-pip) 

This shouldn't be the first time learners clone repos (Unit 2 Workshop onwards requires it of them)